### PR TITLE
Included_HTML_AND_CARD_section

### PR DESCRIPTION
**Pull request by hariFED**
### Changes Made:

1.Created a new HTML document that includes a card component and a font-size demonstration.
2.Added a new CSS document (card.css) specifically for styling the card component.

### Corrections:
1.Adjusted text colour for the description in the Text.css file to enhance readability.
2.Fine-tuned opacity ratios in the style.css file for improved visual aesthetics.

### Additional Information:
1.Verified and tested the changes to ensure proper functionality and styling.
2.Consistently applied coding standards and best practices.
3.Improved the structure of the codebase for better maintainability.

### Reasons for Changes:
1.The new HTML document enhances modularity by separating the card component and font-size demo.
2.Introducing a dedicated card.css file improves code organization and reusability.
3.Addressed readability concerns by adjusting text colour in the Text.css file.
4.Fine-tuned opacity ratios in the style.css file to achieve a more balanced visual appeal.

### Testing:

1.Manually tested the HTML and CSS changes on various screen sizes and browsers.
2.Ensured that the adjustments did not introduce any regressions.

_**Note to Reviewers:**_
Please review the changes thoroughly and provide feedback. Feel free to ask questions or request additional information if needed.

### CODE REVIEW

File: card.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  font-size: var(--description-font-size);
+  font-family: var(--font-family);
+  color: var(--outline-grey);
+  /* border: 1px solid var(--outline-grey); */
+  padding: 40px;
+  gap: 20px;
```
Comment (GREEDnim) [inline card.css:11]: requesting global styles

File: card.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.star {
+  padding-right: 10px;
+}
+
+.star-span {
+  text-decoration: underline dotted 1px;
+  text-underline-offset: 5px;
+}
```
Comment (GREEDnim) [inline card.css]: merge classes :)

File: card.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+.view-details {
+  color: var(--purple);
+  cursor: pointer;
+}
+
+.line {
+  background-color: var(--outline-grey);
+  height: 1px;
+  width: 100%;
```
Comment (GREEDnim) [inline card.css]: width default is 100%

**Comment by GREEDnim** [comment]
requesting to change values in classes as variables.

File: card.css [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  font-size: var(--description-font-size);
+  font-family: var(--font-family);
+  color: var(--outline-grey);
+  /* border: 1px solid var(--outline-grey); */
+  padding: 40px;
+  gap: 20px;
+  /* border-radius: 15px; */
+}
+
+.content-section {
```
Comment (aravind-digitech) [inline card.css]: Task 1 - Add component name prefix (card-content-section)

File: card.css [hunk 5]
```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  font-size: var(--description-font-size);
+  font-family: var(--font-family);
+  color: var(--outline-grey);
+  /* border: 1px solid var(--outline-grey); */
```
Comment (aravind-digitech) [inline card.css]: Remove commented code

File: card.css [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+.star {
+  padding-right: 10px;
+}
+
+.star-span {
+  text-decoration: underline dotted 1px;
+  text-underline-offset: 5px;
+}
+
+.price {
```
Comment (aravind-digitech) [inline card.css]: apply task 1 for all class names

**Review by aravind-digitech — CHANGES_REQUESTED** [review]
pls comment once done

File: card.css (hunk 6 above)
Comment (hariFED) [reply card.css]: Can you explain what is needed to changed? @aravindv97

Comment (aravind-digitech) [reply card.css]: Task 1 - Add component name prefix (card-content-section)

File: card.css (hunk 2 above)
Comment (hariFED) [reply card.css]: **Resolved:** Removed the star class as it has no effect on the div.

File: card.css (hunk 3 above)
Comment (hariFED) [reply card.css]: As the width was not set and the component was not displayed we add a width mannually

File: card.css (hunk 4 above)
Comment (hariFED) [reply card.css]: **Resolved:** Added prefix to all component and committed

File: card.css (hunk 5 above)
Comment (hariFED) [reply card.css]: **Resolved:** Removed the commented code

File: card.css (hunk 6 above)
Comment (hariFED) [reply card.css]: **Resolved:** Added all prefix to the components and deleted unused components.

**Comment by hariFED** [comment]
**Resolved** 
All the issues mentioned above has been solved. @aravindv97 @GREEDnim

**Review by aravind-digitech — CHANGES_REQUESTED** [review]
Looks good

**Review by aravind-digitech — APPROVED** [review]
Looks good

**Pull request merged**